orc8r/cloud/go/services/certifier/servicers/certifier.go

```diff
@@ -449,3 +320,444 @@ func (srv *CertifierServer) CollectGarbageImpl(ctx context.Context) (int, error)
 	}
 	return count, nil
 }
+
+// GetPolicyDecision makes a policy decision when a user attempts to access a resource.
+// For conflicting policy decisions from multiple tokens (e.g. one policy is ALLOW and the other DENY), the DENY effect
+// will take precedent.
+// For resources that do not have any policies addressing it, the policy decision defaults to DENY as well.
+func (srv *CertifierServer) GetPolicyDecision(ctx context.Context, getPDReq *certprotos.GetPolicyDecisionRequest) (*certprotos.PolicyDecision, error) {
```

Comment:
Ahh I need to check my logic here, I don't think it's doing what this docstring is saying bc I changed stuff to support networkid/tenantid scoping...